web-frontend/src/pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx

```diff
@@ -2,15 +2,24 @@ import { Button } from '@massalabs/react-ui-kit';
 import CopyAddress from './CopyAddress';
 import { FiLink } from 'react-icons/fi';
 import Intl from '../../../i18n/i18n';
+import QRCodeReact from 'qrcode.react';
 
 function ReceiveCoins({ ...props }) {
+  const { account } = props;
+  const address = account.address;
+  const formattedAddress = address.slice(0, 4) + '...' + address.slice(-4);
```

Comment:
If we are using or intend to use that in another place why not extract to /utils folder and use it:

```ts
function maskAddress(str: string, mask = ". . .": string): string {
  const start = 3;
  const end = str.length - 3;
  const masked = str.substring(start, end).replace(/./g, mask);

  return str.substring(0, start) + masked + str.substring(end);
}
```
then...
```ts
maskString("blablabla");
```